flink-shaded-hadoop/pom.xml

```diff
@@ -111,6 +111,7 @@ under the License.
 									<include>io.netty:netty:*</include>
 									<include>org.apache.curator:*</include>
 									<include>org.apache.hadoop:*</include>
+									<include>org.mortbay.jetty:*</include>
```

Comment:
This PR does not change that the classes are not hidden. It only puts them into the prepared Hadoop dependency for Flink (which later goes into the Flink fat jar).